reference/entitlement.v1.yaml

```diff
@@ -285,44 +299,51 @@ components:
       required:
         - id
         - name
+      x-tags:
+        - entitlements
     Group:
       title: Group
       type: object
-      description: A sub-group within a school that the entitlement is applicable to.
+      description: A sub-group within a school that the entitlement is applicable to - typically a subject and level.
       properties:
-        standard_subject_code:
+        subjectCode:
           type: string
-          description: Government standard subject code.
-        standard_level:
+          description: 'Officiële elementcode, bron: Ministerie van OCW'
+        level:
           type: string
-          description: Government standard level.
+          description: 'havo, vwo, etc.'
```

Comment:
should include level/year like havo-1, vwo-2, vmbo-gt-3

Reply:
Thanks

Reply:
Actually - just realised this was a rogue addition to my commit - its from entitlements, so will fix it there.